src/components/Input/InputForm.tsx

```diff
@@ -0,0 +1,50 @@
+import CategorySelect from '@/components/Input/CategorySelect'
+import InputField from '@/components/Input/InputField'
+import InputTypeToggle from '@/components/Input/InputTypeToggle'
+import MemoField from '@/components/Input/MemoField'
+import { useInputForm } from '@/hooks/useInputForm'
+
+const InputForm = () => {
+	const {
+		inputType,
+		selectedCategory,
+		categories,
+		errors,
+		register,
+		onSubmit,
+		handleTypeChange,
+		handleCategoryChange,
+	} = useInputForm()
+
+	return (
+		<div className='pt-[50px] pb-[80px] px-10'>
+			<h2 className='text-black text-4xl font-semibold mb-10'>
+				수입 및 지출 입력하기
+			</h2>
+			<form onSubmit={onSubmit}>
+				<InputTypeToggle
+					inputType={inputType}
+					onTypeChange={handleTypeChange}
+				/>
+
+				<div className='flex w-full h-full gap-14 tablet:flex-row'>
```

Comment:
tablet breakpoint에 row를 걸었는데 기본 flex-col을 빼먹은거같아요!
```suggestion
				<div className='flex flex-col w-full h-full gap-14 tablet:flex-row'>
```

Reply:
감사합니다~